control/HTTPCacheControl.php

```diff
@@ -405,7 +408,7 @@ public function applyToResponse($response)
 	 *
 	 * @return string
 	 */
-	public function generateCacheHeader()
+	protected function generateCacheHeader()
```

Comment:
Why did we name this `generateHeaders` rather than `getHeaders`? Not related to your change, just wondering. It's not stateful on the object, and isn't resource intensive - it just gets `$state` in a different representation consistently, which seems like a getter to me.

Reply:
Just following naming convention of `generateCacheHeader`